ignite/templates/app/files/app/params/encoding.go.plush

```diff
@@ -10,7 +10,7 @@ import (
 // This is provided for compatibility between protobuf and amino implementations.
 type EncodingConfig struct {
 	InterfaceRegistry types.InterfaceRegistry
-	Marshaler         codec.Codec
+	Codec             codec.Codec
```

Comment:
This change breaks the scaffolded chains. You need to change the usage of the variable as well

Reply:
Done